### DISCUSSION TITLE
ERROR: couldn't initialize libp2p config: error creating libp2p key: open ~/.lotus/mir.key: no such file or directory

### DISCUSSION THREAD

**Discussion by shanithkk** [filecoin-project/lotus#10900] [Developers/Service Providers]
while initializing the validator  ```./eudico mir validator config init``` throwing error like 
```ERROR: couldn't initialize libp2p config: error creating libp2p key: open ~/.lotus/mir.key: no such file or directory```

**Comment by TippyFlitsUK** [comment]
This is an IPC issue @shanithkk! Please move your question to https://github.com/consensus-shipyard/ipc-actors
Many thanks! 🙏
